domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevmapper/brev/innvilgelsesvp/BeregningMapper.java

```diff
@@ -0,0 +1,177 @@
+                resultat = SelvstendigNæringsdrivende.ny(resultat)
+                        .medNyoppstartet(andel.getNyIArbeidslivet())
+                        .medÅrsinntekt(andel.getBruttoPrÅr().longValue())
+                        .medSistLignedeÅr(getSisteLignedeÅr(andel))
+                        .medInntektLavere_AT_SN(AktivitetStatus.KOMBINERT_AT_SN.equals(andel.getAktivitetStatus()) && dagsatsErNull(andel))
+                        .medInntektLavere_AT_FL_SN(AktivitetStatus.KOMBINERT_AT_FL_SN.equals(andel.getAktivitetStatus()) && dagsatsErNull(andel))
+                        .medInntektLavere_FL_SN(AktivitetStatus.KOMBINERT_FL_SN.equals(andel.getAktivitetStatus()) && dagsatsErNull(andel))
+                        .build();
+            }
+        }
+        return resultat;
+    }
+
+    public static Frilanser mapFrilanser(Beregningsgrunnlag beregningsgrunnlag) {
+        Frilanser resultat = null;
+        for (BeregningsgrunnlagPrStatusOgAndel andel : getAndeler(beregningsgrunnlag)) {
+            if (andel.getAktivitetStatus().erFrilanser()) {
+                resultat = Frilanser.ny(resultat)
+                        .leggTilMånedsinntekt(getMånedsinntekt(andel).longValue())
+                        .build();
+            }
+        }
+        return resultat;
+    }
+
+    // Spesielt for SVP: avkortetPerÅr gir faktisk beregningsgrunnlag for alle arbeidsforhold, og ikke bare arbeidsforholdet det søkes om
+    public static BigDecimal getAvkortetPrÅrSVP(Beregningsgrunnlag beregningsgrunnlag) {
+        return finnFørstePeriode(beregningsgrunnlag).getAvkortetPrÅr();
+    }
+
+    public static boolean erMilitærSivil(Beregningsgrunnlag beregningsgrunnlag) {
+        return beregningsgrunnlag.getAktivitetStatuser().stream()
+                .map(BeregningsgrunnlagAktivitetStatus::aktivitetStatus)
+                .collect(Collectors.toList())
+                .contains(AktivitetStatus.MILITÆR_ELLER_SIVIL);
+    }
+
+    public static boolean inntektOverSeksG(Beregningsgrunnlag beregningsgrunnlag) {
+        return finnFørstePeriode(beregningsgrunnlag).getBruttoPrÅr().compareTo(finnSeksG(beregningsgrunnlag)) > 0;
+    }
+
+    public static BigDecimal finnSeksG(Beregningsgrunnlag beregningsgrunnlag) {
+        return beregningsgrunnlag.getGrunnbeløp().multipliser(6).getVerdi();
+    }
+
+    public static String utledLovhjemmelForBeregning(Beregningsgrunnlag beregningsgrunnlag, Behandling behandling) {
+        boolean revurdering = BehandlingType.REVURDERING.equals(behandling.getBehandlingType());
+        boolean innvilget = BehandlingResultatType.INNVILGET.equals(behandling.getBehandlingsresultat().getBehandlingResultatType());
+        String konsekvensForYtelsen = BehandlingMapper.kodeFra(behandling.getBehandlingsresultat().getKonsekvenserForYtelsen());
+
+        // TODO (ALM) : Fix me! Workaround for å unngå manuelt arbeid for saksbehandlere - Se TFP-1032
```

Comment:
Trenger vi den fortsatt?

Reply:
Jeg tenker kommentaren er fin for å fortelle hvorfor det gjøres, men jeg fjerner "todo/fix"-delen av det siden det neppe er noe som vil bli gjort med det første?